qtp_biom/summary.py

```diff
@@ -28,6 +28,25 @@
 </html>"""
 
 
+def _generate_metadata_file(response, out_fp):
+    """method to minimize code duplication: and merges the prep/sample info files
+
+    Parameters
+    ----------
+    response : dict
+        The response from checking a preparation from Qiita
+    out_fp : str
+        The filepath where we want to store the mer
+    """
+    sf = pd.read_csv(
```

Comment:
These two calls will still interpret missing values as NaN, and also might interpret number-looking strings as numbers is that OK? I usually use this to avoid those problems:

```
pd.read_csv(fn, sep=sep, dtype='str', na_values=[], keep_default_na=True)
```

Reply:
Also, I make sure to set the index after parsing, otherwise some dtype casting might occur.

Reply:
Sounds good. The only comment, `index_col=0` should take care of the index, no?

Reply:
Yes, although my recollection is that setting that during parsing ignores the `dtype=str` argument and can lead to ids being parsed as numbers. Best is to have a separate call like `df.set_index(..., inplace=True)`.